lib/m3u8-parser.js

```diff
@@ -22,6 +27,26 @@ var m3u8parser = module.exports = function() {
       self.emit('item', line.trim());
     }
   }
+  function ytdlParse() {
```

Comment:
A custom function to parse on exit. We could optionally have this literally just be in the event, but this function seemed like the better option. Descriptions of each item in the function below.